packages/neon/neon/lib/src/widgets/error.dart

```diff
@@ -78,56 +95,65 @@ class NeonError extends StatelessWidget {
       color: color,
     );
 
-    final message =
+    final message = Text(
```

Comment:
This widget is only used in 2 of the types. Wouldn't it be better to inline it since it is not very complex?